applications/StructuralMechanicsApplication/tests/test_linear_constraints.py

```diff
@@ -0,0 +1,191 @@
+from __future__ import print_function, absolute_import, division
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import math
+
+def inner_prod(a,b):
+    tmp = 0
+    for i in range(len(a)):
+        tmp += a[i]*b[i]
+    return tmp
+
+class TestLinearConstraints(KratosUnittest.TestCase):
```

Comment:
I would put this to the core, see https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/tests/test_linear_master_slave_constraints.py

Reply:
done